cookbook/deployment/cluster/productionize_cluster.py

```diff
@@ -5,195 +5,251 @@
-
-base
-  The `base directory <https://github.com/flyteorg/flyte/tree/master/kustomize/base>`__ contains minimal configurations for each Flyte component.
-
-dependencies
-  The `dependencies directory <https://github.com/flyteorg/flyte/tree/master/kustomize/dependencies>`__ contains deploy configurations for components like ``PostgreSQL`` that Flyte depends on.
-
-These directories were designed so that you can modify them using ``kustomize`` to generate a custom Flyte deployment.
-In fact, this is how we create the ``sandbox`` deployment.
-
-Understanding the sandbox deployment will help you to create your own custom deployments.
-
-Understanding the Sandbox
-*************************
-
-The sandbox deployment is managed by a set of kustomize `overlays <https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#overlay>`__ that alter the ``base`` configurations to compose the sandbox deployment.
-
-The sandbox overlays live in the `kustomize/overlays/sandbox <https://github.com/flyteorg/flyte/tree/master/kustomize/overlays/sandbox>`__ directory. There are overlays for each component, and a "flyte" overlay that aggregates the components into a single deploy file.
-
-**Component Overlays**
-  For each modified component, there is an kustomize overlay at ``kustomize/overlays/sandbox/{{ component }}``.
-  The overlay will typically reference the ``base`` for that component, and modify it to the needs of the sandbox.
-
-  Using kustomize "patches", we add or override specific configs from the ``base`` resources. For example, in the "console" overlay, we specify a patch in the `kustomization.yaml <https://github.com/flyteorg/flyte/blob/master/kustomize/overlays/sandbox/flyte/kustomization.yaml>`__. This patch adds memory and cpu limits to the console deployment config.
-
-  Each Flyte component requires at least one configuration file. The configuration files for each component live in the component overlay. For example, the FlyteAdmin config lives at `kustomize/overlays/sandbox/flyte/admin/deployment.yaml <https://github.com/flyteorg/flyte/blob/master/kustomize/overlays/sandbox/flyte/admin/deployment.yaml>`__. These files get included as Kubernetes configmaps and mounted into pods.
-
-**Flyte Overlay**
-  The ``flyte`` overlay is meant to aggregate the components into a single deployment file.
-  The `kustomization.yaml overlay <https://github.com/flyteorg/flyte/blob/master/kustomize/overlays/sandbox/flyte/kustomization.yaml>`__ in that directory lists the components to be included in the deploy.
-
-  We run ``kustomize build`` against the ``flyte`` directory to generate the complete `sandbox deployment yaml <https://github.com/flyteorg/flyte/blob/master/deployment/sandbox/flyte_generated.yaml>`__ we used earlier to deploy Flyte sandbox.
-
-Creating Your Own Deployment
+In order to handle production load robustly, securely and with high availability, there's a number of important tasks that need to
```

Comment:
@SandraGH5 do you mind proof-reading this doc and suggesting edits as you see fit?

Reply:
Sure.